tox.ini

```diff
@@ -1,20 +1,16 @@
 [tox]
 envlist =
     lint
-    py{27,34,35,36}-django111
-    py{34,35,36,37}-django200
-    py{35,36,37}-django210
-    py{35,36,37}-django220
-    py{36,37,38}-django300
+    py{36,37,38,39}-django220
+    py{36,37,38,39}-django300
+    py{36,37,38,39}-django310
 
 [travis]
 python =
-  2.7: py27
-  3.4: py34
-  3.5: py35
   3.6: py36
-  3.7: py37, coverage, lint
```

Comment:
This should be adapted in the codecov gh action. But i don't think it should block this PR.